std_msgs/Header header
kimera_interface/ClassInstance[] instances
